server.js

```diff
@@ -1,233 +1,12 @@
-const _ = require('lodash');
-const jwt = require('jwt-simple');
-const express = require('express');
-const app = express();
-
-const homebrewApi = require('./server/homebrew.api.js');
-const GoogleActions = require('./server/googleActions.js');
-const serveCompressedStaticAssets = require('./server/static-assets.mv.js');
-
-app.use('/', serveCompressedStaticAssets(`${__dirname}/build`));
+require('app-module-path').addPath(__dirname);
```

Comment:
Can you clarify what this package does for us? Do we need to introduce another dependency for this PR to work?

Reply:
The problem I encountered is that we have two entry points to our app - a regular one through the existing main file, which is being launched under `node` and the new one which is being introduced with tests.

When you launch test, you basically just execute a `.js` file which contains test using `node`, but under some  different context/environment, i.e. all include paths are different comparing to a regular launch.

That `app-module-path` is my attempt to align things. If you have any other suggestions/ideas on how it can be fixed - I'm all for it